nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -27,6 +27,9 @@
   This also allows configuring runtime settings of AMDVLK and enabling experimental features.
 - The `moonlight-qt` package ([Moonlight game streaming](https://moonlight-stream.org/)) now has HDR support on Linux systems.
 
+- [Sched-ext](https://github.com/sched-ext/scx), a Linux kernel feature to run schedulers in userspace, is now available [services.scx](options.html#opt-services.scx.enable).
+  This can ramp up performance in certain workloads like gaming. Requires running a linux kernel of version 6.12 or later.
```

Comment:
"ramp up" -> "improve", but maybe just remove that entire sentence unless you have strong data to back it up.

"Requires ~~running~~ a Linux (capitalization) kernel ~~of~~ version 6.12 or later."

Reply:
> but maybe just remove that entire sentence unless you have strong data to back it up.

I don't have the patience to generate graphs nor I will look for benchmarks. But I observed low to none lag when browsing my system, watching my videos, all while compiling a kernel for instance. Again, this was my on my own setup.

Upstream claims this though, with a [video reference](https://github.com/sched-ext/scx?tab=readme-ov-file#sched_ext-schedulers-and-tools).

Reply:
I suggest we err on the side of caution here, and don't promise anything specific, especially on a feature that's largely still experimental.